.meta/sinks/kafka.toml

```diff
@@ -13,12 +13,12 @@ write_to_description = "[Apache Kafka][urls.kafka] via the [Kafka protocol][urls
 tls_options = ["+enabled", "-verify"]
 
 [sinks.kafka.options.bootstrap_servers]
-type = "[string]"
+type = "string"
 common = true
-examples = [["10.14.22.123:9092", "10.14.23.332:9092"]]
+examples = ["10.14.22.123:9092,10.14.23.332:9092"]
```

Comment:
This is still an array :smile:

Reply:
@lukesteensen I thought its an array to let you show more examples, in the rendered doc it should be a single string?

Reply:
Ha. One day I'll understand how these files work...